From 5ff2b59266e67eba88fb235560c083b0b4f0d6ce Mon Sep 17 00:00:00 2001
From: James Cameron <quozl@laptop.org>
Date: Sat, 30 Sep 2017 19:03:24 +1000
Subject: [PATCH] Fix choose an image button

When a new instance of ImageViewer is started, the "Choose an image"
does not work.  A traceback is in log;

Traceback (most recent call last):
  File "/usr/share/sugar/activities/ImageViewer.activity/ImageViewerActivity.py", line 396, in _show_picker_cb
    if not self._want_document:
AttributeError: 'ImageViewerActivity' object has no attribute '_want_document'

Regression caused by d496604 which removed the initialisation of the
attribute from the object.
---
 ImageViewerActivity.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ImageViewerActivity.py b/ImageViewerActivity.py
index c7bf7b3..3e5baf0 100644
--- a/ImageViewerActivity.py
+++ b/ImageViewerActivity.py
@@ -95,6 +95,7 @@ def __init__(self, handle):
         # Status of temp file used for write_file:
         self._tempfile = None
         self._close_requested = False
+        self._want_document = True
 
         self._zoom_out_button = None
         self._zoom_in_button = None
